src/hooks/useActivitiesLocalStorage.js

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from 'react'
+
+export default function useActivitiesLocalStorage() {
+  const initialActivities = () =>
+    JSON.parse(localStorage.getItem('myActivities') || '[]')
+  const [activities, setActivities] = useState(initialActivities)
+
+  useEffect(() => {
```

Comment:
Brauchst du hier den useEffect mit der Bedingung dass er nur ausgeführt wird, wenn sich activities ändert?  Wird der ganze useActivitiesLocalStorage Hook nicht eh nur dann aufgerufen, wenn sich sich activities ändert? Sonst würde er sich ja in dieser Funktion doppelt aufrufen, also nach der letzten Zeile nochmal? (Das kann man sich bestimmt gut im Debugger angucken ob er eh immer ausgeführt wird, wenn der Hook ausgeführt wird, ich sach dir, ich bin ein Fan :) )

Reply:
Werde ich so ausprobieren. Wollte da ja eh nochmal ran. :-)